index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portofolio</title>
+</head>
+
+<body>
+
+    <header>
+        <div class="header">
+            <div class="mylogo"><a class="logo">My Logo</a></div>
```

Comment:
- [ ] Kindly remove the div and leave it in a tag
because of convention, semantics, and user experience(UX).